### PR TITLE
Fixed the meta description tag in the teacher app micro frontend

### PR DESCRIPTION
**Pull request by sidtohan**
## Description

For SEO, the meta description tag in the teacher-app micro frontend was changed from => 
"Web site created using create-react-app" 
**to** 
"The teacher app Micro Frontend used in the Shiksha app.
It can also function as a standalone package."

### Changes

List the changes done to fix a bug or introducing a new feature.
In the index.html file of the public folder of teacher-app, changed the meta description tag.

## How to test

The app may be run individually or as whole. And then the meta tag may be inspected using the developer tools in any browser of choice.

### CODE REVIEW

File: packages/teacher-app/public/index.html [hunk 1]
```diff
@@ -7,7 +7,8 @@
     <meta name="theme-color" content="#000000" />
     <meta
       name="description"
-      content="Web site created using create-react-app"
+      content="The teacher app Micro Frontend used in the Shiksha app.
```
Comment (ChakshuGautam) [inline packages/teacher-app/public/index.html:10]: ~~Shiksha app~~ => Shiksha Platform

**Comment by ChakshuGautam** [comment]
I think we should do this for other modules as well @sidtohan

**Comment by sidtohan** [comment]
Ok sir I will do the same for others also

**Comment by sidtohan** [comment]
Should I make a new PR or continue on this one only?

**Pull request merged**